src/components/Games/Number/NumberMain.js

```diff
@@ -47,28 +60,43 @@ const Input = styled.input`
 `;
 
 export const NumberMain = () => {
-  const [num, setNum] = useState(0);
+  const [num, setNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
+  const [newNum, setNewNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
 
   function RandomNumberInRange(min, max) {
     return Math.floor(Math.random() * (max - min +1)) + min;
   };
 
-  const handleClick = () => {
-    setNum(RandomNumberInRange(1, 999999999));
+  const handleClick = (e) => {
+    setNewNum(RandomNumberInRange(1, 9));
+    setNum(String(num) + String(newNum)); 
```

Comment:
`setNum(num + RandomNumberInRange(0,9))` If you make `RandomNumberInRange` return a string instead of int then you won't have to keep converting. You can just keep everything as a string because you only ever need it as that.